openlibrary/templates/account/mybooks.html

```diff
@@ -1,41 +1,33 @@
 $def with (user, mybooks, key=None, owners_page=False, public=False, counts=None, lists=None, component_times={})
```

Comment:
Why is this not taking in a mybooks object? A lot of these parameters are things that exist on the my books object.